### PR TITLE
ACP2E-1598: [Documentation] Update GraphQL documentation to reflect changes introduced in ACP2E-1494

### PR DESCRIPTION
**Pull request by bubasuma**
## Description

New concrete type `RoutableUrl` was added that implements `RoutableInterface` and inherits the following fields with no additional fields.
```
relative_url
redirect_code
type
```

Document to be updated:
https://developer.adobe.com/commerce/webapi/graphql/schema/products/interfaces/routable/

## Related Issue

[ACP2E-1494](https://jira.corp.adobe.com/browse/ACP2E-1494): Url Rewrite to external URL does not work on GraphQL

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have signed the [Adobe Open Source CLA](https://opensource.adobe.com/cla.html).
- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Review by jhadobe — APPROVED** [review]

File: src/pages/graphql/schema/products/interfaces/routable.md [hunk 1]
```diff
@@ -183,3 +184,39 @@ The following query returns information about the specified URL key. The query c
...
 }
 ```
+
+## Routable URL
+
+`RoutableUrl` is the default implementation of RoutableInterface. This type is returned when the URL is not linked to an entity.
+
+### Example
+
+In the following example, an internal URL `support.html` is configured to redirect to an external URL `https://support.example.com/` using URL Rewrite.
```
Comment (keharper) [inline src/pages/graphql/schema/products/interfaces/routable.md:194]: Add a sentence explaining why `type` contains a null value.

Comment (bubasuma) [reply src/pages/graphql/schema/products/interfaces/routable.md]: Good point. I added it in an info box right before the example section. Should I move it down here under Example, please let me know!

File: src/pages/graphql/schema/products/interfaces/routable.md [hunk 2]
```diff
@@ -183,3 +184,43 @@ The following query returns information about the specified URL key. The query c
   }
 }
 ```
+
+## Routable URL
+
+`RoutableUrl` is the default implementation of RoutableInterface. This type is returned when the URL is not linked to an entity.
```
Comment (keharper) [inline src/pages/graphql/schema/products/interfaces/routable.md:190]:
```suggestion
`RoutableUrl` is the default implementation of RoutableInterface. This type is returned when the URL is not linked to a product or CMS page or to a category. As a result, the `RoutableUrl.type` field always returns `null`.
```

Delete the InlineAlert note box.

**Review by keharper — COMMENTED** [review]
Would you add a sentence that describes

**Review by keharper — APPROVED** [review]

**Pull request merged**